arbitrary-users-patch/build_images.sh

```diff
@@ -36,4 +37,8 @@ while read -r line; do
     echo "Pushing ${NAME_FORMAT}/${base_image_name}:${TAG}" to remote registry
     docker push "${NAME_FORMAT}/${base_image_name}:${TAG}" | cat
   fi
+  BUILT_IMAGES="${BUILT_IMAGES}    ${NAME_FORMAT}/${base_image_name}:${TAG}\n"
 done < "${SCRIPT_DIR}"/base_images
+
+echo "Built images:"
+echo -e "$BUILT_IMAGES"
```

Comment:
I'm not sure these changes are required, is it leftover from the development process?

Reply:
I added them during the dev process to aid testing (I looped over all images to ensure the tools installed correctly), but I left it in because I thought it'd be useful for CI build logs. I'm not attached to them, though, so would be fine removing. At the end of the run, it prints out
```
Built images:
    quay.io/eclipse/che-cpp-rhel7:nightly
    quay.io/eclipse/che-dotnet-2.2:nightly
    quay.io/eclipse/che-dotnet-3.1:nightly
    quay.io/eclipse/che-golang-1.12:nightly
    quay.io/eclipse/che-golang-1.14:nightly
    quay.io/eclipse/che-java11-gradle:nightly
    quay.io/eclipse/che-java11-maven:nightly
    quay.io/eclipse/che-java8-maven:nightly
    quay.io/eclipse/che-nodejs10-community:nightly
    quay.io/eclipse/che-nodejs10-ubi:nightly
    quay.io/eclipse/che-nodejs8-centos:nightly
    quay.io/eclipse/che-php-7:nightly
    quay.io/eclipse/che-python-3.6:nightly
    quay.io/eclipse/che-python-3.7:nightly
    quay.io/eclipse/che-quarkus:nightly
    quay.io/eclipse/che-rust-1.39:nightly
```
which is easier to read than grepping through the output to find `Successfully tagged quay.io/eclipse/xxxx`

Reply:
Agreed, sounds good to me